sql/moz-fx-data-shared-prod/firefox_desktop_derived/urlbar_events_daily_v1/metadata.yaml

```diff
@@ -0,0 +1,19 @@
+friendly_name: Urlbar Events Daily
+description: |-
+  Each row in this table represents a daily summary statistic of impressions
+  and clicks per product result type per a host of meaningful dimensions.
+owners:
+- tbrooks@mozilla.com
```

Comment:
one minor thing: if @alekhyamoz or someone from DE would like to be the owner, feel free to update this. didn't know who to have here at the time